Sources/Components/HappinessRating/HappinessRatingView.swift

```diff
@@ -106,24 +116,67 @@ public class HappinessRatingView: UIView {
     }
 }
 
+extension HappinessRatingView: RatingImageViewDelegate {
+    func ratingImageViewText(for rating: HappinessRating) -> String? {
+        switch rating {
+        case .angry:
+            return "Veldig irriterende"
```

Comment:
Guess the delegate method mentioned above should be called from here, so we can get rid of these strings

Reply:
Yep, was supposed to have these in the demo and call the delegate from here.